roles/spark-standalone-install/meta/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+dependencies:
+  - { role: hadoop-variables, when: hadoop_profile == 'without-hadoop' }
```

Comment:
I'd rather keep the yaml syntax consistent wherever possible.

Does this work?
```yaml
    - role: hadoop-variables
      when: hadoop_profile == "without-hadoop"
...
```